nautobot_golden_config/forms.py

```diff
@@ -236,19 +236,31 @@ class Meta:
         fields = "__all__"
 
 
+class ComplianceFeatureBulkEditForm(TagsBulkEditFormMixin, NautobotBulkEditForm):  # pylint: disable=too-many-ancestors
```

Comment:
Something went amok here.

Reply:
If makes sense to re-order so you don't have to continue with this, makes sense.

Reply:
It doesn't matter what we do on this branch, now that the baked branch is correct. That is why changes we make here will "stick".